Stackup-Rwanda/knights-bn-frontend
# #170947603 Create signup page

#### What does this PR do?
- Adds signup page.
#### Description of Task to be completed?
- Design UI
- connect to the API
#### How should this be manually tested?
- run `git clone https://github.com/Stackup-Rwanda/knights-bn-frontend.git` to have the repo locally.
- run `cd /knights-bn-frontend.git` to get inside the project directory.
- run `git checkout -t origin/ft-signup-170947603` to switch to this branch.
- run `npm i` to install all dependencies.
- run `npm dev-start` to start the server.
- the browser should start by itself to the homePage.
- type in `http://localhost:8080/signup` to get the signup page
- populate in the required fields and hit `SIGN UP`
- after clicking on the `account verification link` via your email make sure to change the redirecting URL in the URL path to match the local branch: 
```
ex: http://localhost:8080/
```

#### What are the relevant pivotal tracker stories?
[link](https://www.pivotaltracker.com/story/show/170947603)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/30593186/80951820-e153e900-8df8-11ea-9ed5-60aca3d2d23f.png)

![image](https://user-images.githubusercontent.com/30593186/80958992-321e0e80-8e06-11ea-939e-397e636d9490.png)

![image](https://user-images.githubusercontent.com/30593186/80959240-ad7fc000-8e06-11ea-8668-bad62796b5fb.png)

### The email verification link is sent after correct data validation.
![image](https://user-images.githubusercontent.com/30593186/81052376-994dc880-8ec3-11ea-81e5-8f3a13032ca4.png)
